src/api.py

```diff
@@ -43,15 +43,16 @@ def get_results(
     values = []
     for query_param in query_params:
         value = kwargs.get(query_param)
+        if value is None:
+          continue
```

Comment:
Nice.

Reply:
Oh thanks for commenting on this specific line because I just realized it was pasted weird and the indentation is off